### PR TITLE
Add setup instructions for Carthage

### PR DESCRIPTION
**Pull request by Palleas**
Cf. https://github.com/Carthage/Carthage

### CODE REVIEW

**Comment by robb** [comment]
FWIW, it's already compatible, so adding

```
github "ResearchKit/ResearchKit" "master"
```

to your Cartfile is enough to get started.

**Comment by Palleas** [comment]
I figured targeting "stable" would be better since there is no tag (yet?).

**Comment by robb** [comment]
Good point, did not see that branch.

File: README.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ Or, for the latest changes, use the `master` branch:
 git clone https://github.com/ResearchKit/ResearchKit.git
 ```
 
+You can also use [Carthage](https://github.com/Carthage/Carthage) by adding the following 
+instructions in your [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#cartfile) and running `carthage update`:
+
+````
+github "ResearchKit/ResearchKit" "stable"
+````
```
Comment (remiprev) [inline README.md]: Picky comment, but code fences are delimited by three `\`’s, not four :smile:

Comment (Palleas) [inline README.md]: Fixed, thx!

**Comment by bavarious** [comment]
Not sure what Apple’s policy is (will be?) regarding badges in README.md, but projects can also use [this badge](https://github.com/Carthage/Carthage#declare-your-compatibility) to declare they’re compatible with Carthage.

**Comment by jwe-apple** [comment]
Thanks for this suggestion. We're thinking of putting dependency management on a separate page, linked from the readme, rather than cluttering up the main page with it.

Holding this open to track.

**Comment by Palleas** [comment]
Let me know if I can help with this page! :+1:

**Comment by jwe-apple** [comment]
This has already been done via PR #5

**Pull request closed without merging**